src/components/selectable/selectable_list/selectable_list.tsx

```diff
@@ -61,6 +61,11 @@ export type EuiSelectableOptionsListProps = CommonProps &
      * The default content when `true` is `↩ to select/deselect/include/exclude`
      */
     onFocusBadge?: EuiSelectableListItemProps['onFocusBadge'];
+    /**
+     * Use virtualized rendering for list items with `react-window`.
+     * Requires that each option be the same height.
+     */
+    isVirtualized?: boolean;
```

Comment:
```suggestion
    /**
     * Use virtualized rendering for list items with `react-window`.
     * Sets each row's height to the value of `rowHeight`.
     */
    isVirtualized?: boolean;
```

I'm also wondering if we need to set `rowHeight` to optional since it won't be applied if `isVirtualized=false`.